crypto/sha/sha512.c

```diff
@@ -18,16 +18,16 @@
 /*-
  * IMPLEMENTATION NOTES.
  *
- * As you might have noticed 32-bit hash algorithms:
+ * As you might have noticed, 32-bit hash algorithms:
  *
  * - permit SHA_LONG to be wider than 32-bit
  * - optimized versions implement two transform functions: one operating
- *   on [aligned] data in host byte order and one - on data in input
+ *   on [aligned] data in host byte order, and one - on data in input
```

Comment:
If we're going to be adding a comma into this sentence, let's address the more egregious grammar - the use of the dash here.

Best replace the dash with the word it is substituting, `operating`, to give this (might need re-formatting)

```suggestion
 *   on [aligned] data in host byte order, and one operating on data in input
```

Reply:
That is an excellent addition. It greatly improves the readability of the section. Thank you!